src/ui/ui-kit/Stack/Stack.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { HStack } from '../ui-kit/HStack';
-import { VStack } from '../ui-kit/VStack';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { VStack } from 'src/ui/ui-kit/VStack';
 
 export const Stack = React.forwardRef<
```

Comment:
Shared, used in a few places and looks convenient, so I extracted it and moved to the ui-kit

Reply:
Also absolute imports are good for mocks inside ui-lab. So I agree with such changes :))